extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -253,8 +253,8 @@ export const postDeployScriptFriendlyName = localize('postDeployScriptFriendlyNa
 
 // Build
 
-export const NetCoreInstallationConfirmation: string = localize('sqlDatabaseProjects.NetCoreInstallationConfirmation', "The .NET Core SDK cannot be located. Project build will not work. Please install .NET Core SDK version 3.1 or update the .NET Core SDK location in settings if already installed.");
-export function NetCoreSupportedVersionInstallationConfirmation(installedVersion: string) { return localize('sqlDatabaseProjects.NetCoreSupportedVersionInstallationConfirmation', "Currently installed .NET Core SDK version is {0}, which is not supported. Project build will not work. Please install .NET Core SDK version 3.1 or update the .NET Core SDK supported version location in settings if already installed.", installedVersion); }
+export const NetCoreInstallationConfirmation: string = localize('sqlDatabaseProjects.NetCoreInstallationConfirmation', "The .NET Core SDK cannot be located. Project's extension may not load and build will not work. Please install .NET Core SDK version 3.1 or update the .NET Core SDK location in settings if already installed.");
```

Comment:
was the extension not loading pre-existing behavior? from what I remember it was loading and only if extension is loaded user can attempt and fail during build.

Reply:
Yes, it wasn't loading prior to my latest changes as well.